etc/kayobe/kolla/config/monasca/notification_templates/slack_template.j2

```diff
@@ -0,0 +1 @@
+{% raw %}{{ alarm_name }} has triggered on: Host {{ metrics[0].dimensions.hostname }}:{% endraw %}  http://{{ilab_vip_address}}{% raw %}:3000/plugins/monasca-app/page/alarms?dimensions=hostname:{{ metrics[0].dimensions.hostname }}{% endraw %}
```

Comment:
One problem here is that if the alarm fires on multiple hosts, depending on how the alarm is configured, then you might mask a failure on all but one host. You could work around it by having a policy of creating an alarm for each host (I think you can do this with --match-by hostname). Another option is to not mention the hostname here and link to the top level Monasca Grafana App page and from there you will get a global status. There is also the option of a for loop.

Reply:
Valid points. Unlikely to be a problem with the current set of alarms since they all match by hostname but this ought to be future proof. 
Will strip out the hostname.